wayback-webapp/src/main/webapp/WEB-INF/replay/Toolbar.jsp

```diff
@@ -284,7 +287,8 @@ function trackMouseMove(event,element) {
        </td>
        <td style="padding:0!important;">
        <a style="position:relative; white-space:nowrap; width:<%= imgWidth %>px;height:<%= imgHeight %>px;" href="" id="wm-graph-anchor">
```

Comment:
I think there will be no difference whether it has been added inside the yearChart or outside it. The behavior is the same. The functionality of the element with id="wm-graph-anchor" is just highlighting the year when mouse pointer is hovering over it.

I am afraid of not fully understand the strange behavior.